packages/builder/src/components/backend/DataTable/modals/ManageAccessModal.svelte

```diff
@@ -94,7 +94,7 @@
   loadDependantInfo()
 </script>
 
-<ModalContent showCancelButton={false} confirmText="Done">
+<ModalContent showCancelButton={false} showConfirmButton={false}>
```

Comment:
This button does nothing, and it can give a false impression of an action not being persisted. The update on the dropdowns trigger the "save"

https://github.com/user-attachments/assets/570b65e6-b624-4491-b07f-5012f439b0ab